rails/benchmarks/bm_activerecord_create.rb

```diff
@@ -4,10 +4,7 @@
 
 require_relative 'support/benchmark_rails.rb'
 
-ActiveRecord::Base.establish_connection(
-  adapter: 'sqlite3',
-  database: ':memory:'
-)
+ActiveRecord::Base.establish_connection(ENV['DATABASE_URL'])
```

Comment:
@tgxworld I already fixed it here https://github.com/kirs/ruby-bench-suite/commit/2e3ca512d7afbfa6a1b35597ab5e3f6b16b74a65